std/file.d

```diff
@@ -862,7 +862,7 @@ unittest
 version(StdDdoc) void getTimesWin(R)(R name,
                                   out SysTime fileCreationTime,
                                   out SysTime fileAccessTime,
-                                  out SysTime fileModificationTime)
+                                  out SysTime fileModificationTime) @safe
```

Comment:
Isn't this misleading, since the real version of the function (below) doesn't have the `@safe` attribute?